docker/environment.yml

```diff
@@ -41,7 +40,7 @@ dependencies:
   - cfgrib
   - pydap
   - cartopy >= 0.20.1
-  - descartes
+  - descartes # Is this really needed???
```

Comment:
Yes `descartes` is needed, reason in this commit description https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/commit/90ccbece93469c3783e8d786592ac92e7fdf5f15

Reply:
We really need to remove this dependency. There hasn't been a new release in nearly 6 years! Will look into updating that notebook.